### PR TITLE
SEO-177279-Page-With-Redirect-WF

### PR DESCRIPTION
**Pull request by berylouma**
Hello,

I have worked on links with redirects.

Regards,
Beryl.

### CODE REVIEW

File: WindowsForms/DataGrid/Merging.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ documentation: ug
...
+The [QueryCoveredRangeEventArgs](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.DataGrid.SfDataGrid.html#Syncfusion_WinForms_DataGrid_SfDataGrid_QueryCoveredRange) of the QueryCoveredRange event provides information about the cell which triggered this event. The [QueryCoveredRangeEventArgs.OriginalSender](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.DataGrid.Events.GridEventArgs.html#Syncfusion_WinForms_DataGrid_Events_GridEventArgs_OriginalSender) returns the `SfDataGrid` which fired this event for `DetailsView`. By using the [QueryCoveredRangeEventArgs.Range](https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.DataGrid.Events.QueryCoveredRangeEventArgs.html#Syncfusion_WinForms_DataGrid_Events_QueryCoveredRangeEventArgs_Range) property, the adjacent cells can be merged.
```
Comment (Yvone-Atieno) [inline WindowsForms/DataGrid/Merging.md:15]: Update this link under QueryCoveredRangeEventArgs
https://help.syncfusion.com/cr/windowsforms/Syncfusion.WinForms.DataGrid.Events.QueryCoveredRangeEventArgs.html

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

**Review by Yvone-Atieno — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**